index.js

```diff
@@ -0,0 +1,88 @@
+/// DOM Elements ///
+const playerDiv = document.querySelector('div.player')
+const playerImg = document.querySelector('img')
+const playerName = document.querySelector('h2')
+const playerNickName = document.querySelector('em')
+const playerLikes = document.querySelector('p.likes')
+const playerGoals = document.querySelector('ul#goals')
+
+const playerGoalForm = document.querySelector('#new-goal-form')
+
+
+const likeBtn = document.querySelector('.like-button')
+
+// console.log(likeBtn)
+/////////////////////// Deliverable 1 /////////////////////////////
+function intialize() {
+    fetch('http://localhost:3000/players/1')
+    .then(response => response.json())
+    .then(playerData => {
+        playerImg.src = playerData.photo,
+        playerName.textContent = playerData.name,
+        playerNickName.textContent = playerData.nickname,
+        playerLikes.textContent = `${playerData.likes} Likes`,
+        playerData.goals.forEach(goal => renderGoals(goal))
+    })
+}
+intialize()
+
+function renderGoals(goal) {
+    const li = document.createElement('li')
+    const pElement = document.createElement('p')
+    const aElement = document.createElement('a')
+    pElement.textContent = goal.description
+    aElement.href = goal.link
+    aElement.textContent = goal.link
+    li.append(pElement, aElement)
+    playerGoals.append(li)
+}
+//////////////////////////////////////////////////////////////////
+
+///////////////////////// Deliverable 2 /////////////////////////
+
+function updateLikes() {
+    fetch('http://localhost:3000/players/1', {
+        method: 'PATCH',
+        headers: {'Content-Type': "application/json"},
+        body: JSON.stringify({
+            likes: parseInt(playerLikes.textContent) + 1
+        }),
+    })
+    .then(response => response.json())
+    .then(playerData => playerLikes.textContent = `${playerData.likes} Likes`)
+}
+
+likeBtn.addEventListener("click", updateLikes)
+
+//////////////////////////////////////////////////////////////////
+
+///////////////////////// Deliverable 3 /////////////////////////
+
+playerGoalForm.addEventListener("submit", submitGoal)
+
+function submitGoal(event) {
+    event.preventDefault()
+
+    const oneGoal = {
+        // Not sure how to assign this :( 
+        // playerId: event.target.playerID.dataset.id,
```

Comment:
You do need the `playerId` in this object, so you've got the right idea. One way of doing this would be to assign a `data-id` to the form element directly, for example in your initialize function:

```js
playerGoalForm.dataset.id = playerdata.id
```

Then you can use it here (making sure to parseInt since the API expects a number):


```js
const oneGoal = {
   playerId: parseInt(event.target.dataset.id),
```